### PR TITLE
VB-314 Make extra API call to get convicted status on profile page

### PR DESCRIPTION
**Pull request by tpmcgowan**
Also restricts viewing of offenders to those in Hewell

### CODE REVIEW

File: server/services/prisonerProfileService.test.ts [hunk 1]
```diff
@@ -25,6 +26,23 @@ describe('Prisoner profile service', () => {
     })
 
     it('Retieves and processes data for prisoner profile with visit balances', async () => {
```
Comment (psoleckimoj) [inline server/services/prisonerProfileService.test.ts:28]: I know it's not in the change but spelling 😁

**Review by psoleckimoj — DISMISSED** [review]
LGTM

File: server/services/prisonerProfileService.test.ts (hunk 1 above)
Comment (tpmcgowan) [reply server/services/prisonerProfileService.test.ts]: well spotted 🤦‍♂️!

**Review by psoleckimoj — APPROVED** [review]
lgtm

**Pull request merged**